### PR TITLE
Add aria-labels to XFA form elements. (bug 1723422)

### PR DESCRIPTION
**Pull request by brendandahl**

### CODE REVIEW

**Review by calixteman — APPROVED** [review]
LGTM

**Comment by brendandahl** [comment]
I'm looking through some more files, but it looks like we may want to fallback to tooltip if speak isn't available.

File: src/core/xfa/template.js [hunk 1]
```diff
@@ -201,6 +201,22 @@ function setTabIndex(node) {
...
+    return null;
+  }
+  const assist = field.assist;
+  if (assist.speak && assist.speak[$content] !== "") {
+    return assist.speak[$content];
+  }
+  if (assist.toolTip) {
+    return assist.toolTip[$content];
+  }
+  // TODO: support finding the relatated caption element. See xfa_bug1718037.pdf
```
Comment (Snuffleupagus) [inline src/core/xfa/template.js:215]: Spelling-nit: `relatated` -> `related`

**Pull request merged**